knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml

```diff
@@ -1103,55 +1297,10 @@ spec:
     - name: https-webhook
       port: 443
       targetPort: 8443
+    - name: http-metrics
+      port: 9090
+      targetPort: 9090
   selector:
     app: kafka-webhook-eventing
 
 ---
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  labels:
-    kafka.eventing.knative.dev/release: devel
-  name: config-kafka-leader-election
-  namespace: knative-eventing
-  annotations:
-    knative.dev/example-checksum: "96896b00"
-data:
-  _example: |
-    ################################
-    #                              #
-    #    EXAMPLE CONFIGURATION     #
-    #                              #
-    ################################
-
-    # This block is not actually functional configuration,
-    # but serves to illustrate the available configuration
-    # options and document them in a way that is accessible
-    # to users that `kubectl edit` this config map.
-    #
-    # These sample configuration options may be copied out of
-    # this example block and unindented to be in the data block
-    # to actually change the configuration.
-
-    # leaseDuration is how long non-leaders will wait to try to acquire the
-    # lock; 15 seconds is the value used by core kubernetes controllers.
-    leaseDuration: "15s"
-
-    # renewDeadline is how long a leader will try to renew the lease before
-    # giving up; 10 seconds is the value used by core kubernetes controllers.
-    renewDeadline: "10s"
-
-    # retryPeriod is how long the leader election client waits between tries of
-    # actions; 2 seconds is the value used by core kubernetes controllers.
-    retryPeriod: "2s"
-
-    # buckets is the number of buckets used to partition key space of each
-    # Reconciler. If this number is M and the replica number of the controller
-    # is N, the N replicas will compete for the M buckets. The owner of a
-    # bucket will take care of the reconciling for the keys partitioned into
-    # that bucket.
-    buckets: "1"
-  leaseDuration: "15s"
-  renewDeadline: "10s"
-  retryPeriod: "2s"
-  map-lease-prefix.kafka-broker-controller.knative.dev.eventing-kafka-broker.control-plane.pkg.reconciler.source.Reconciler: kafka-controller.knative.dev.eventing-kafka.pkg.source.reconciler.source.reconciler
```

Comment:
Why is this deleted?

/hold

Reply:
@pierDipi why is there none for this, in upstream ?